docs/how-to/download-real-time-sources.rst

```diff
@@ -0,0 +1,116 @@
+How to download Real-time Ubuntu kernel source code
+===================================================
+
+The source code for Ubuntu's real-time kernel is available for download.
+The following guide will show you how to download it for your system's release of Ubuntu.
+The process depends on the release of Ubuntu you are using.
+
+Enable access to the real-time Ubuntu kernel
+---------------------------------------------
+
+.. tab-set::
+    :sync-group: release
+
+    .. tab-item:: LTS releases (22.04, 24.04, etc.)
+        :sync: lts
+
+        .. note::
+            LTS releases of Ubuntu require Ubuntu Pro to download their respective real-time kernel source code.
+            See the `Ubuntu Pro documentation <https://documentation.ubuntu.com/pro/>`_ for more information.
```

Comment:
Shouldn't this point to the particular how-to?
https://documentation.ubuntu.com/real-time/en/latest/how-to/enable-real-time-ubuntu/